modules/Microsoft.Sql/servers/databases/deploy.bicep

```diff
@@ -110,6 +110,31 @@ param diagnosticMetricsToEnable array = [
   'WorkloadManagement'
 ]
 
+@description('Optional. Specifies the mode of database creation.')
+@allowed([
+  'Default'
+  'Copy'
+  'OnlineSecondary'
+  'PointInTimeRestore'
+  'Recovery'
+  'Restore'
+  'RestoreLongTermRetentionBackup'
+  'Secondary'
+])
+param createMode string = 'Default'
+
+@description('Optional. Resource ID of database if createMode set to Copy, Secondary, PointInTimeRestore, Recovery or Restore.')
+param sourceDatabaseId string = ''
```

Comment:
```suggestion
param sourceDatabaseResourceId string = ''
```

Reply:
The property is named sourceDatabaseId in the API. Ref: https://learn.microsoft.com/en-us/azure/templates/microsoft.sql/servers/databases?pivots=deployment-language-bicep

It's perfectly fine for me to change it if that is the style you'd like to use going forward, I just see there is some inconsistency in the repo when it comes to naming of parameters containing resource IDs.